requirements.txt

```diff
@@ -7,6 +7,7 @@ django-csp==3.7 # For Content Security Policy
 django-db-file-storage==0.5.5 # Support for storage in the database
 django-mysql==4.7.0
 django-tinymce==3.4.0 # Rich text editor
+django-watchman==1.3
```

Comment:
I saw from the [Docs](https://github.com/mwarkentin/django-watchman#logging) of this library that we can add the watchman to the Django logging configuration. But this PR doesn't account for that, but I do see some level of watchman logging in OpenShift pod.  

Why did you opt out of not adding the watchman logging config? Either way is fine, just wanted to understand your rationale

Reply:
I didn't see this in MyLA, so I guess I wasn't too worried about it. I added it though at the `DEBUG` level. It's pretty noisy, there may be some template issues with `watchman`, but they don't seem to affect functionality.